### DISCUSSION TITLE
Discussion To Documentation

### DISCUSSION THREAD

**Discussion by DahyeonWoo** [euphony-io/euphony#246] [Ideas]
I want to write several documents, but I want to manage them with one issue. I wonder if it's possible to allocate an issue to me.

**Comment by yeonns** [comment]
@DahyeonWoo How about writing one document first? :smile: It can help us to review documentation easily and after that you can add more documentation quickly! What do you want to write first?

**Comment by DahyeonWoo** [comment]
@yeonns I would like to make a contribution to organizing the contents listed in the `knowledge share` category of dicussion.
Starting with charset documentation , I will try to make it as much as possible, such as fft and modem.
Charset documentation will also be posted on discusion first.
In conclusion, the first thing to make is charset documentation!

**Reply by yeonns** [reply]
@DahyeonWoo I got it! Then, I will make charset document issue and assign it to you :+1:
